great_expectations/compatibility/aws.py

```diff
@@ -0,0 +1,29 @@
+from __future__ import annotations
+
+from great_expectations.compatibility.not_imported import NotImported
+
+AWS_NOT_IMPORTED = NotImported(
+    "Unable to load AWS connection object; please 'pip install boto3 botocore'"
+)
+
+try:
+    import boto3
+except ImportError:
+    boto3 = AWS_NOT_IMPORTED
+
+try:
+    import botocore
+except ImportError:
+    botocore = AWS_NOT_IMPORTED
+
+try:
+    from botocore.exceptions import ClientError
```

Comment:
Do you need ParamValidationError here too? And to set ParamValidationError = AWS_NOT_IMPORTED wherever ClientError is being set to that?

Reply:
Saw this too 😅 PR has been updated.